core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -733,7 +733,7 @@ else if (Tables.getTableState(context.getInstance(), table) == TableState.OFFLIN
     void queueMutations(final MutationSet mutationsToSend) throws InterruptedException {
       if (null == mutationsToSend)
         return;
-      binningThreadPool.execute(new Runnable() {
+      binningThreadPool.execute(Trace.wrap(new Runnable() {
```

Comment:
Oh, that's lame. You're totally right -- I thought the former is how it worked. Maybe I'm remembering pre-HTrace? What you have sounds great then :)